notebooks/openvino/requirements.txt

```diff
@@ -1,6 +1,4 @@
-git+https://github.com/openvinotoolkit/nncf.git@develop
```

Comment:
@vuiseng9, was asking about the ability to install NNCF from the source. This is required for audio models as far as I understood. @vuiseng9, can you comment please?

Reply:
Never mind, this comment related to the samples only.